src/main/java/com/exasol/repository/ReleaseLetterParser.java

```diff
@@ -0,0 +1,107 @@
+package com.exasol.repository;
+
+import static com.exasol.ReleaseRobotConstants.DATE_REGEX;
+import static com.exasol.ReleaseRobotConstants.VERSION_REGEX;
+
+import java.time.LocalDate;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import com.exasol.repository.ReleaseLetter.Builder;
+
+public final class ReleaseLetterParser {
+    public static final String TICKET_NUMBER_REGEX = "#[1-9]\\d*\\b";
+
+    private ReleaseLetterParser() {
+        // prevent instantiation
+    }
+
+    /**
+     * Create a new {@link ReleaseLetter} from a string.
+     *
+     * @param fileName name of the release letter file
+     * @param content file's content as a string
+     */
+    public static ReleaseLetter parseReleaseLetterContent(final String fileName, final String content) {
+        final Builder builder = ReleaseLetter.builder(fileName);
```

Comment:
Afterwards save this as a instance variable.
Add a `parse()` method.

This means that each time you want to parse something new, you construct a new parser, but that's perfectly okay. Also you probably only do that for the last release letter anyway.

Reply:
Sorry, I didn't get this suggestion.  `parse()` method for what class? And why do I need to save the builder as an instance?

Reply:
As discussed via call.